### ISSUE TITLE
Touch prop not working on first run of a tooltip showing when using delegate addon

### ISSUE THREAD

**Issue by jshah4517** [atomiks/tippyjs#707]
## Bug description

We are using the event delegate addon as we wish to set tippy to run on all elements with a title attribute, even those that are generated dynamically after the page load.

That works well, but we have since tried to add the touch prop for handling the tooltips better on mobile, and it seems to be ignored on the first run of the tooltip showing but works on clicking after that. The reproduction must be ran on a mobile device, I have tested it in Android Chrome.

## Reproduction

Delegate CodePen link: https://jsfiddle.net/2dmb4kx1/1/

Open this in a mobile browser and click on the string, the tooltip will show. If you click away and then click on it again, it will not show unless you hold the click for 500ms. It should be expected that this functionality happens on the first click also.

Normal CodePen link: https://jsfiddle.net/70dmrbgx/

The normal case (without delegate addon) works as expected.

**Comment by jshah4517** [comment]
Hey @atomiks, sorry known you've been busy with the new release, had any chance to check over this? It seems to be happening on the new version too.

V6 delegate CodePen link: https://jsfiddle.net/u6eoLvsn/
V6 normal CodePen link: https://jsfiddle.net/u6eoLvsn/1/

I can take a screencast of the issue if it helps.

**Comment by atomiks** [comment]
It didn't happen on Chrome mobile DevTools iirc so it would be difficult to verify a fix. You're free to send a PR with a solution though

**Comment by jshah4517** [comment]
I've just been looking in to this more carefully now. It is replicable in mobile dev tools, here's a screencast to show what the issue is: http://somup.com/cYe02y1VeZ

I made the delay longer to more clearly demonstrate the problem, it shows the tooltip instantly on the first click, and then respects the delay on the second and further clicks. 

I've deduced the problem is down to this code inside the delegate function. If you comment out the `showOnCreate` it seems to work as it should.
```
var childProps = Object.assign({}, nativeProps, {
  showOnCreate: true
});
```

I wasn't sure what this line was for though, if it could be removed or if `showOnCreate` prop just needs to consider the `touch` prop value.
